customer-config/checkout-app/cha.settings.v1.json

```diff
@@ -98,6 +102,11 @@
         "kiosk"
       ],
       "properties": {
+        "schedule": {
```

Comment:
I'm wondering is we should move this under sco ? we dont support non SCO schedule?